core/app/models/concerns/spree/default_price.rb

```diff
@@ -13,6 +13,13 @@ module DefaultPrice
         autosave: true
     end
 
+    # Returns `#prices` prioritized for being considered as default price
+    #
+    # @return [Array<Spree::Price>]
```

Comment:
This should be a `ActiveRecord::Relation`.

Reply:
Right x 3, & fixed!! :slightly_smiling_face: